src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person,
+}) => (
+  <section className="Person">
```

Comment:
it's a good practice to use object destructing
```suggestion
export const Person = ({ person }) => {
  const { name, age, ... } = person;
  
  return (
    <section className="Person">
```